workbench-core/datasets/src/iamHelper.ts

```diff
@@ -64,7 +64,8 @@ export class IamHelper {
     const returnDoc = new PolicyDocument();
     _.forEach(policyObj.Statement, (s) => {
       if (s.Sid === targetSid) {
-        if (!_.isArray(s.Principal.AWS)) throw new Error('Cannot remove principal since only one principal is assigned');
+        if (!_.isArray(s.Principal.AWS))
+          throw new Error('Cannot remove principal since only one principal is assigned');
```

Comment:
```suggestion
        if (!_.isArray(s.Principal.AWS)) {
          throw new Error('Cannot remove principal since only one principal is assigned');
        }
```